### PR TITLE
[#1190] [Jyoti] Add codegen for nodeJs Unirest support

### PR DESCRIPTION
**Pull request by guptajyoti845**

### CODE REVIEW

File: helpers/codegen/codegen.js [hunk 1]
```diff
@@ -48,6 +49,7 @@ export const codegens = [
   SalesforceApexCodegen,
   ShellHTTPie,
   WgetCodegen,
+  NodeUnirestRequestCodegen,
```
Comment (liyasthomas) [inline helpers/codegen/codegen.js:52]: Change `NodeUnirestReq`  to `NodeJsUnirestReq`.
Also we keep alphabetical ordering in this array of codegens. So move unirest right next to `NodeJsNativeCodegen`.

Comment (liyasthomas) [reply helpers/codegen/codegen.js:52]: Also rebase to latest `master` branch.

**Comment by liyasthomas** [comment]
@guptajyoti845 do you need any help?

File: helpers/codegen/codegen.js (hunk 1 above)
Comment (guptajyoti845) [reply helpers/codegen/codegen.js]: @liyasthomas above mentioned feedbacks are encorporated.

**Comment by liyasthomas** [comment]
@guptajyoti845 can you run `npm run test` to update snapshots and push it.

**Comment by guptajyoti845** [comment]
> @guptajyoti845 can you run `npm run test` to update snapshots and push it.

@liyasthomas 
updated the snapshot test

**Review by liyasthomas — APPROVED** [review]
LGTM.

**Pull request merged**